Sources/MessagingPush/PushHandling/iOSPushEventListener.swift

```diff
@@ -11,9 +11,6 @@ protocol PushEventListener: AutoMockable {
     // return nil if the push was not handled by CIO SDK.
     // return true if should diplay the push in foreground.
     func shouldDisplayPushAppInForeground(_ push: PushNotification) -> Bool?
-
-    func newNotificationCenterDelegateSet(_ newDelegate: UNUserNotificationCenterDelegate?)
-    func beginListening()
```

Comment:
Looks like there is no use of `UserNotifications` in `iOSPushEventListener` class anymore. If I understood it right then do we still need to import `UserNotifications` in this file ?

https://github.com/customerio/customerio-ios/blob/3b66a9e5087b6cabd2154a57f80e08f095df94f9/Sources/MessagingPush/PushHandling/iOSPushEventListener.swift#L4

Reply:
👍 https://github.com/customerio/customerio-ios/pull/470/commits/f3a543cf9e23ca475d04c42b6e40b3a416a6baf8